python-requirements.in

```diff
@@ -2,6 +2,9 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
+# package management
+poetry
```

Comment:
Actually this should probably be a dev dependency not a runtime dep. I'll remove this in later commits.